README.md

```diff
@@ -48,60 +66,62 @@ $ cmake ..
-* *VS 2013* with the november 2013 CTP (corrective patch)  should work, but is untested.
-[https://www.microsoft.com/en-us/download/confirmation.aspx?id=41151]
  
 #### Dependencies
 
 *Qt* distributes version 5.6 with precompiled libraries for VS 2015 - 64 bits. 
 If using earlier versions of Qt (5.4 or 5.5)  or a different toolset you may have to compile Qt yourself.
 You will probaby have to manually point cmake to the Qt folder (see Troubleshooting below)
 
-Eigen : included as a submodule in the git repository.
-Assimp : included as a submodule in the git repository.
-glbinding : included as a submodule in the git repository.
-
-#### Building
+Other dependencies (Eigen, Assimp and glbinding) are included as a submodule in the git repository.
 
-As long as cmake run smoothly the engine should compile.
+#### Getting started with Visual Studio
 
-### Run
+Thanks to the integrated support of CMake in Visual Studio, you don't need a VS solution to build your project: open the Radium folder (via *File* > *Open* > *Folder ...* or `devenv.exe <foldername>`).
+VS should run cmake, generate the target builds (Debug and Release by default).
 
-* Don't forget to copy the third party DLLs in the executable folder :
- * Qt libraries (Qt5xxx.dll or Qt5xxxd.dll if you are in debug) : Core, Gui and Widgets
-* Set "radium" as your startup project
-* Change the application working directory to `$(OutDir)..` (go to the "radium" project properties, *Debugging* menu, *Working Directory*) to get the shaders to load properly.
-
-## Building on Windows with QtCreator / MinGW
+You may have Cmake errors occuring at the first run (see Troubleshooting section below).
+To fix them, you need to edit the VS-specific file `CMakeSettings.json`, via *CMake* > *Change CMake Settings* > path-to-CMakeLists (configuration-name) from the main menu. 
+For instance, it usually requires to set cmake build types manually, and to give path to Qt libraries.
+To fix it, edit `CMakeSettings.json`, such that
+```
+      {
+        ...
+        "cmakeCommandArgs": "-DCMAKE_PREFIX_PATH=C:\\Qt\\5.7\\winrt_x64_msvc2015 -DCMAKE_BUILD_TYPE=Debug",
+        ...
+      },
+      {
+        "name": "x64-Release",
+        ...
+        "cmakeCommandArgs": "-DCMAKE_PREFIX_PATH=C:\\Qt\\5.7\\winrt_x64_msvc2015 -DCMAKE_BUILD_TYPE=Release",
+        ...
+      }
+```
 
-### Building
+#### Compilation
 
-You can now run Cmake in Qt Creator. It should locate Qt libs automatically. 
-It should also find GLEW provided you followed the previous step. 
-If not, you can always set `GLEW_INCLUDE_DIR` and `GLEW_LIBRARY` manually to their correct locations.
+Right click on CMakeList.txt > build > all.
 
-### Running
+#### Run
 
-If you run the program within QtCreator the path should be correctly set, and the executable should run fine.
+Assimp and GlBinding libs need to be manually copied in the bin directory (FIXME).
```

Comment:
I would suggest "DLLs" instead of "libs" (in windows world :  `.lib` = static library)